packages/query/src/graphql/plugins/GetMetadataPlugin.ts

```diff
@@ -12,6 +12,8 @@ const {version: packageVersion} = require('../../../package.json');
 const indexerUrl = argv('indexer') as string | undefined;
 
 const METADATA_TYPES = {
+  name: 'string',
```

Comment:
no need to expose them in query service

Reply:
oops, let me reread through the diff, there's probably a bit like this